### PR TITLE
Re-style tag clouds

### PR DESCRIPTION
**Pull request by osteele**
In consultation with Jeff:

* Added the closest matching icon from the Ionic framework.
* Unselected tags use the tag color
* [Roundrects](https://www.folklore.org/StoryView.py?story=Round_Rects_Are_Everywhere.txt)!

## Required
Changes must conform to these requirements:
* [x] `yarn test` passes.  All new and existing tests pass.
* [x] `yarn lint` passes. All new code follows the code style of this project.

## Aspirational
We don't yet require these, but they are nice to have:
* [n/a] New code is covered by new or existing tests.
* [x] Changed code is covered by new or existing tests.

### CODE REVIEW

File: src/components/label-pane.jsx [hunk 1]
```diff
@@ -60,7 +78,7 @@ LabelPane.propTypes = {
     showUnselected: PropTypes.bool,
     possibleLabels: PropTypes.object,
     selectedLabels: PropTypes.array,
-    contentClass: PropTypes.string,
+    contentClass: PropTypes.string
```
Comment (kylecombes) [inline src/components/label-pane.jsx]: I like to use dangling commas for the reasons outlined [here](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8). I also believe it's part of the Airbnb spec.

File: src/components/label-pane.jsx [hunk 2]
```diff
@@ -69,5 +87,5 @@ LabelPane.defaultProps = {
     editable: true,
     selectedLabels: [],
     showUnselected: true,
-    contentClass: '',
+    contentClass: ''
```
Comment (kylecombes) [inline src/components/label-pane.jsx]: Another omitted dangling comma...

File: src/components/label-pane.jsx (hunk 1 above)
Comment (osteele) [reply src/components/label-pane.jsx]: Agreed. My editor should be adding these not removing them. I'll look into why it's working backwards.

Comment (osteele) [reply src/components/label-pane.jsx]: I’ll fix these after #159 is in dev, so I can do it systematically.

**Pull request merged**